ops/ccs-ops-misc/load_test/high_volume_suite.py

```diff
@@ -0,0 +1,374 @@
+        self.run_task_by_parameters(
+            base_path="/v2/fhir/ExplanationOfBenefit",
+            params={
+                "_lastUpdated": f"gt{self.last_updated}",
+                "patient": self.bene_ids.pop(),
+                "_IncludeTaxNumbers": "true",
+                "_format": "application/fhir+json",
+            },
+            name="/v2/fhir/ExplanationOfBenefit search by id / lastUpdated / includeTaxNumbers",
+        )
+
+    @tag("eob", "eob_test_id", "v2")
+    @task
+    def eob_test_id(self):
+        """Explanation of Benefit search by ID"""
+        self.run_task_by_parameters(
+            base_path="/v2/fhir/ExplanationOfBenefit",
+            params={"patient": self.bene_ids.pop(), "_format": "application/fhir+json"},
+            name="/v2/fhir/ExplanationOfBenefit search by id",
+        )
+
+    @tag("patient", "patient_test_coverage_contract", "v2")
+    @task
+    def patient_test_coverage_contract(self):
+        """Patient search by Coverage Contract, paginated"""
+
+        def make_url():
+            contract = self.contract_data.pop()
+            return create_url_path(
+                "/v2/fhir/Patient",
+                {
+                    "_has:Coverage.extension": f'https://bluebutton.cms.gov/resources/variables/ptdcntrct01|{contract["id"]}',
+                    "_has:Coverage.rfrncyr": f'https://bluebutton.cms.gov/resources/variables/rfrnc_yr|{contract["year"]}',
+                    "_count": 25,
+                    "_format": "json",
+                },
+            )
+
+        self.run_task(
+            name="/v2/fhir/Patient search by coverage contract (all pages)",
+            headers={"IncludeIdentifiers": "mbi"},
+            url_callback=make_url,
+        )
+
+    @tag("patient", "patient_test_hashed_mbi", "v2")
+    @task
+    def patient_test_hashed_mbi(self):
+        """Patient search by hashed MBI, include identifiers"""
+
+        def make_url():
+            return create_url_path(
+                "/v2/fhir/Patient/",
+                {
+                    "identifier": f"https://bluebutton.cms.gov/resources/identifier/mbi-hash|{self.hashed_mbis.pop()}",
+                    "_IncludeIdentifiers": "mbi",
+                },
+            )
+
+        self.run_task(
+            name="/v2/fhir/Patient search by hashed mbi / includeIdentifiers = mbi",
+            url_callback=make_url,
+        )
+
+    @tag("patient", "patient_test_id_include_mbi_last_updated", "v2")
+    @task
+    def patient_test_id_include_mbi_last_updated(self):
+        """Patient search by ID with last updated, include MBI"""
+        self.run_task_by_parameters(
+            base_path="/v2/fhir/Patient",
+            params={
+                "_id": self.bene_ids.pop(),
+                "_format": "application/fhir+json",
+                "_IncludeIdentifiers": "mbi",
+                "_lastUpdated": f"gt{self.last_updated}",
```

Comment:
Yeah, this is still using the existing method of calculating `last_updated` via `data.py`'s `get_last_updated()`. Basically, it seems like `self.last_updated` will always be 2 weeks prior to the current date.

This is something that should be removed from `BFDUserBase` and handled per-suite depending on their needs.

@mharvey perhaps usages of `self.last_updated` in this new load suite should be replaced with some specific date like Logan's suggested? To be fair, I'm not sure _which_ date that should be, I think @lsmitchell would know better than I. I can handle removing `last_updated` from `BFDUserBase` and the required changes to the regression suite in a future ticket.

Reply:
yeah for either test its probably fine to use 1-3 weeks before 2022-07-11 (test and see how much comes back from that i guess) since thats when the last records were created/loaded, and im assuming thats the date that gets used for the record's last updated date.

Reply:
I handled this by overriding the value for `self.last_updated` in the `__init__` for `HighVolumeUser`